src/ray/core_worker/core_worker.cc

```diff
@@ -1462,13 +1462,13 @@ std::string CoreWorker::OverrideTaskOrActorRuntimeEnv(
     std::vector<std::string> *runtime_env_uris) {
   std::shared_ptr<rpc::RuntimeEnv> parent = nullptr;
   if (options_.worker_type == WorkerType::DRIVER) {
-    if (serialized_runtime_env == "") {
+    if (serialized_runtime_env == "{}") {
```

Comment:
Is it possible to be `""` as well in some cases? Should we check both?

Reply:
Good point, I only observed `"{}"` but it's probably better to be safe and also check `""`.  I've updated the PR.